src/pages/SignIn.js

```diff
@@ -43,15 +42,9 @@ export default class SignIn extends Component {
         }
       }, [
         div({ style: { maxWidth: 900 } }, [
-          div({ style: { display: 'flex', marginBottom: '1rem', alignItems: 'center' } }, [
-            div({ style: { fontWeight: 500, marginRight: '2rem' } }, [
-              div({ style: { fontSize: 40, color: colors.slate } }, ['Welcome to']),
-              div({ style: { fontSize: 80, color: colors.gray[0] } }, ['TERRA'])
-            ]),
-            logo({ size: 265 })
-          ]),
-          div({ style: { fontSize: 40, fontWeight: 500, color: colors.slate } }, ['New User?']),
-          div({ style: { fontSize: 20, marginBottom: '2rem' } }, ['Terra requires a Google Account.']),
+          div({ style: { fontSize: 54, marginBottom: '1.5rem', color: colors.green[0] } }, ['Welcome to Terra']),
```

Comment:
The mocks has the logo moved above "Welcome to Terra", what's the reason for removing it entirely?

Reply:
Bringing this inline with the newer mocks.

Reply:
I'm looking at this one, not sure if there is a newer mock: 
https://projects.invisionapp.com/d/main#/console/16427254/343450766/preview 
It still has the logo, just moved and a different size. Maybe this can be another place where we put the conditional logo?

Reply:
Aha! That was apparently changed Friday, I'll tweak...